tribits/common_tpls/FindTPLHDF5.cmake

```diff
@@ -37,6 +37,13 @@ IF (HDF5_ALLOW_PREFIND)
 
   FIND_PACKAGE(HDF5 COMPONENTS ${HDF5_COMPNENTS})
 
+  # Make sure that HDF5 is parallel.
```

Comment:
Indeed. EpetraExt uses the respective symbols only for `HAVE_MPI`, e.g., https://github.com/trilinos/trilinos/blob/master/packages/epetraext/src/inout/EpetraExt_HDF5.cpp#L347.